### PR TITLE
Automation tests for faq update conversation language change and assignee change

### PR DESCRIPTION
**Pull request by prateek-kommunicate**
1. [ConversationTest.kt](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/compare/development...CM-2218-Automation-Tests-for-FAQ-Update-Conversation-Language-Change-and-Assignee-Change-Android-SDK?expand=1#diff-c9d9cab4a9138ba4c409976cd312984e03a9793050d79e00601526878cd777e3)

a. `testUpdateConversationAssigneeAndVerifyFromDashboard`
b. `testUpdateConversationTeamIdAndVerifyFromDashboard`
c.  `testConversationWithLanguageChange`
d.   `testChangeAssigneeFromDashboardAndVerifyStatusOnSDK`

2. [FAQTest.kt](https://github.com/Kommunicate-io/Kommunicate-Android-Chat-SDK/compare/development...CM-2218-Automation-Tests-for-FAQ-Update-Conversation-Language-Change-and-Assignee-Change-Android-SDK?expand=1#diff-6758f0125a7b39f253af56cc2466a329cd2dec557a423503b6e2f92a6ce54a2b)

a. `testLaunchConversationAndFAQVisibility`

### CODE REVIEW

File: app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt [hunk 1]
```diff
@@ -260,6 +264,223 @@ class ConversationTest {
...
     }
 
+    @Test
+    fun testUpdateConversationAssigneeAndVerifyFromDashboard() {
+        val tempUser = getRandomKmUser()
+        val latch = CountDownLatch(1)
+        var groupId = 0
+        val botIds = listOf("inline-code-34rpc")
+        val updateBotAssigneeId = "kk-3s8r3"
+        val agentIds = listOf("prateek.singh@kommunicate.io")
```
Comment (AbhijeetRanjan308) [inline app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt:274]: Don't push the email or other details like this create a Utility class and store these variables there.

File: app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt [hunk 2]
```diff
@@ -260,6 +264,223 @@ class ConversationTest {
...
+
+    @Test
+    fun testUpdateConversationTeamIdAndVerifyFromDashboard() {
+        val tempUser = getRandomKmUser()
+        val latch = CountDownLatch(1)
+        var groupId = 0
+        val botIds = listOf("inline-code-34rpc")
+        val initialTeamId = "103785933"
+        val updateTeamId= "106336264"
+        val agentIds = listOf("prateek.singh@kommunicate.io")
```
Comment (AbhijeetRanjan308) [inline app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt:338]: similar

File: app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt [hunk 3]
```diff
@@ -260,6 +264,223 @@ class ConversationTest {
...
+        onView(isRoot())
+            .perform(waitForLatch(latch))
+
+        sendMessageAsUser(getRandomString())
+
+        runBlocking {
+            val response = chatAPI.transferChatToBotUser(
+                token = chatAuthToken,
+                groupId = groupId.toString(),
+                assigneeId = "prateek.singh@kommunicate.io"
```
Comment (AbhijeetRanjan308) [inline app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt:455]: similar

**Review by AbhijeetRanjan308 — CHANGES_REQUESTED** [review]

**Review by AbhijeetRanjan308 — APPROVED** [review]

File: app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt (hunk 1 above)
Comment (prateek-kommunicate) [reply app/src/androidTest/java/kommunicate/io/sample/ConversationTest.kt:274]: As discussed: I will be different test cases.

**Pull request merged**